DeratControl.Ifrastructure/Repositories/UserRepository.cs

```diff
@@ -14,5 +17,10 @@ public UserRepository(DbContext databaseContext)
         {
 
         }
+
+        public bool Exists(string email)
+        {
+            return databaseContext.Set<User>().Where(x => x.Email == email).FirstOrDefault() != null;
```

Comment:
return databaseContext.Set<User>().Where(x => x.Email == email).FirstOrDefault() != null;
this line can be substituded by databaseContext.Set(User)().AnyAsync(x => x.Email == email)